test/views/EditSetRecordEditDiscardSpec.scala

```diff
@@ -57,35 +55,33 @@ class EditSetRecordEditDiscardSpec extends BaseSpec {
         )
       )
       document must haveBackLink("/edit-set/1/record/EditRecordOciTest/edit", "edit-set.record.save.back")
-
     }
 
-    "render the header" in {
-
-      val document = generateDocument()
+    "render the header" in new TestCase {
```

Comment:
I believe that one of the implicit requirements of this ticket is to keep the exact scenarios, so this is likely out of scope, but: why do we have a separate scenario just to verify the header?

Reply:
I'm not sure, but I think it was added when the Logout link was added to sign in pages. Lots of the view specs have a "render the header" test.

Reply:
OK. Perhaps it will be reviewed later.

Irrelevant to this ticket.